exchanges/account/account.go

```diff
@@ -0,0 +1,69 @@
+package account
+
+import (
+	"errors"
+	"sync"
+
+	"github.com/thrasher-corp/gocryptotrader/currency"
+)
+
+var accounts = make(map[string]*Holdings)
```

Comment:
This is more of a question. Should we be storing account information like this? Storing it is good for when its not changing, but then once it changes, we don't ever want to refer to something that's outdated in case we put down an order which we don't have funds for. Perhaps when an event such as place order occurs, we trigger an update of our holdings? Or maybe a "lastupdated" field? Not sure, but the only holdings we can 100% trust is one retrieved from an API call

Reply:
This will be continually updated via a synchronisation agent. I have something started in the synchronisation update I have in one of my branches. Need to also add a dispatch mux to this as well.

Reply:
I can only review these PRs as they are unless they are a draft, linked to another PR, the PR descriptions says this is dependent on something else or that it is incomplete. You have checked `Any dependent changes have been merged and published in downstream modules` so I assumed this had nothing else coming in. I can not know your intention for these other updates you refer to

Reply:
Okay I will be more specific on description let me update.